hikari/api/rest.py

```diff
@@ -326,6 +326,114 @@ async def delete_channel(
             'Public Server Updates' channel cannot be deleted.
         """
 
+    @abc.abstractmethod
+    async def edit_my_voice_state(
+        self,
+        guild: snowflakes.SnowflakeishOr[guilds.PartialGuild],
+        channel: snowflakes.SnowflakeishOr[channels_.PartialChannel],
+        *,
+        suppress: undefined.UndefinedOr[bool] = undefined.UNDEFINED,
+        request_to_speak: undefined.UndefinedOr[bool] = undefined.UNDEFINED,
+    ) -> None:
+        """Edit the current user's voice state in a stage channel.
+
+        Parameters
+        ----------
+        guild : hikari.snowflakes.SnowflakeishOr[hikari.guilds.PartialGuild]
+            Object or Id of the guild to edit a voice state in.
+        channel : hikari.snowflakes.SnowflakeishOr[hikari.channels.TextChannel]
+            Object or Id of the channel to edit a voice state in.
+
+        Other Parameters
+        ----------------
+        suppress : hikari.undefined.UndefinedOr[builtins.bool]
+            If defined, whether the user should be allowed to become a speaker
+            in the target stage channel.
```

Comment:
The name of the argument seems to contradict what the argument actually controls

Reply:
If you want to also rename it on voice stqtef tfhen go ahead but it literally changes the suppresfs field

Reply:
I know, but the description makes no sense.

```suggestion
        suppress : hikari.undefined.UndefinedOr[builtins.bool]
            If specified, whether the user should not be allowed to become a speaker
            in the target stage channel.
```

Reply:
Either description makes as much sense as the other, unless you specifically state the behaviour of true vs false